xdsl/dialects/riscv.py

```diff
@@ -433,7 +433,7 @@ def custom_print_attributes(self, printer: Printer) -> Set[str]:
 
 
 AssemblyInstructionArg: TypeAlias = (
-    AnyIntegerAttr | LabelAttr | SSAValue | IntRegisterType | str
+    AnyIntegerAttr | LabelAttr | SSAValue | IntRegisterType | str | int
```

Comment:
`int` is already supported, turns out, not sure why it's not an option in this union